Towny/src/main/java/com/palmergames/bukkit/towny/command/ResidentCommand.java

```diff
@@ -347,6 +352,19 @@ private void parseResidentPlotlist(Player player, String[] split) throws TownyEx
 		TownyMessaging.sendPlotList(player, res, page, total);
 	}
 
+	private void parseResidentOutlawlist(Player player, String[] split) throws TownyException {
+		checkPermOrThrow(player, PermissionNodes.TOWNY_COMMAND_RESIDENT_OUTLAWLIST.getNode());
+
+		Resident resident;
+		if (split.length == 0) {
+			resident = getResidentOrThrow(player);
+		} else {
+			resident = getResidentOrThrow(split[0]);
+		}
+
+		TownyMessaging.sendMessage(player, TownyFormatter.getFormattedTownyObjects(Translatable.of("outlaws").forLocale(player), new ArrayList<>(resident.getTownsOutlawedIn())));

```

Comment:
`Translatable.of("outlaws")` A new lang string should be added to the en-US.yml file which uses `Outlawed In:` as a value.

Reply:
It's probably not necessary to have resident.getTownsOutlawedIn() wrapped in a new ArrayList.

Reply:
> It's probably not necessary to have resident.getTownsOutlawedIn() wrapped in a new ArrayList.

the method expects a list of TownyObjects and that seems to fix it, i got that from a similar method that also puts it in a new arraylist